src/NServiceBus.AcceptanceTesting/AcceptanceTestingTransport/AcceptanceTestingTransportInfrastructure.cs

```diff
@@ -0,0 +1,146 @@
+﻿namespace NServiceBus
```

Comment:
This class isn't public, so namespace should be `NServiceBus.AcceptanceTesting`.